tests/test_jenkins.py

```diff
@@ -118,17 +111,12 @@ async def request(*args, **kwargs):  # pylint: disable=unused-argument
 
     retry = {'total': 5, 'statuses': [HTTPStatus.INTERNAL_SERVER_ERROR]}
 
-    try:
-        jenkins = Jenkins(get_host(), get_user(), get_password(), retry=retry)
-
+    async with Jenkins(get_host(), get_user(), get_password(), retry=retry) as jenkins:
```

Comment:
Why did you change it?

Reply:
Functionally-speaking, it's identical (since either way the `.close()` is being called after the `with` block vs the `try` block), but this this usage then follows the best practice of using an async context manager to auto-close resources.

If it's significant to do so, I can revert it with no issues

Reply:
We need both tests, as it was and with context manager

Reply:
Done, re-added them back